### ISSUE TITLE
$interpolateProvider - StartSymbol & EndSymbol

### ISSUE THREAD

**Issue by floscom** [daniel-nagy/md-data-table#70]
The **StartSymbol** and **EndSymbol** from the **$interpolateProvider** are not used.

**Comment by daniel-nagy** [comment]
Can you be more specific as to why this is an issue?  The default is to use double curly brackets.

**Comment by floscom** [comment]
Yeah i know. I am using this $interpolateProvide to change the Symbols to [[ & ]] because of some issues with my template engine.

```
var app = ...
app.config(function($interpolateProvider) {
            $interpolateProvider.startSymbol('[[');
            $interpolateProvider.endSymbol(']]');
});
```

and on line 13 in the file md-data-table.js you find this:

```
template.text('{{ name }}');
```

i've never used this .text function before, therefore i have never issues with custom symbols.

**Comment by daniel-nagy** [comment]
Ah okay, I need to get the start and stop symbols from the `$interpolateProvider` instead of assuming they will be the double curly brackets.

**Comment by floscom** [comment]
no big deal ;)
thumbs up for this plugin, i like it!

**Comment by daniel-nagy** [comment]
I just released v0.8.1 to fix this issue. Thanks! I'm happy to hear you like it.
